configure.ac

```diff
@@ -562,6 +576,20 @@ fi
 
 AC_ARG_WITH(unixodbc,
 AS_HELP_STRING([--with-unixodbc=DIR], [build odbc driver against unixODBC in DIR]))
+
+if test "x$with_unixodbc" = "xyes"; then
+	# The user asked for unixodbc support, but didn't tell us
+	# where to look. We'll try to get the installation prefix from
+	# the odbc_config executable, and if we can't, we'll display
+	# an error. We require odbc_config to be in $PATH.
+	ODBC_INC=`odbc_config --include-prefix`
```

Comment:
this is just the directory, should also add the -I option.
I honestly won't change unixODBC.
I hope in a future they'll add support for pkg-config too.